stdlib/websub/src/main/ballerina/websub/commons.bal

```diff
@@ -526,6 +525,16 @@ public type WebSubHub object {
     # + return - `error` if an error occurred with unregistration
     public function unregisterTopic(string topic) returns error?;
 
+    # Retrieves an array of topics which registered in the Hub.
+    #
+    # + return - An array of available topics
+    public extern function getAvailableTopics() returns string[];
+
+    # Retrieves an array of registered subscriber details for a given topic.
+    #
+    # + topic - The subscriber details required topic
+    # + return - An array of subscribers
+    public extern function getTopicSubscribers(string topic) returns SubscriberDetails[];
```

Comment:
Shall we change `getTopicSubscribers` to `getSubscribers`? IMO, topic is redundant?

Also do we need the `an array` bit in the description, since we have it in the docs for the return value? Shall we say
```ballerina
     # Retrieves details of subscribers registered to receive updates for a particular topic.
     #
     # + topic - The topic for which details need to be retrieved
     # + return - An array of subscriber details
```